core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -130,6 +134,20 @@ public MultiSearchResponse(Item[] items) {
     public Item[] getResponses() {
         return this.items;
     }
+    
+    /**
+     * How long the msearch took.
+     */
+    public TimeValue getTook() {
```

Comment:
The method exists in both `SearchResponse` and `BulkResponse`.
I would prefer to keep the consistency unless there is a good reason no to.